.github/workflows/ks-py-coding-style.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             ${{ runner.os }}-
 
       - name: Install dependencies
-        run: pip install -r backend/kesaseteli/requirements.txt -r backend/kesaseteli/requirements-dev.txt
+        run: cd backend/kesaseteli && pip install -r requirements.txt -r requirements-dev.txt
```

Comment:
Is this change about style or does it fix something?

Reply:
I can answer it, `requirement.txt` has a relative path to a `shared` folder, and since `pip` has a bug that will translate this relative path to an incorrect path (it calculates the path from the folder where we run the command, not the actual location of the `requirement.txt`) This is a workaround way to solve that issue

Reply:
It helps when installing the shared package so that it will be found correctly from the relative path: `-e ../shared`

Reply:
ok!